Plugins/DD4hep/include/Acts/Plugins/DD4hep/DD4hepDetectorStructure.hpp

```diff
@@ -39,8 +39,9 @@ class DD4hepDetectorStructure {
   struct Options {
     /// The log level of the tools
     Logging::Level logLevel = Logging::INFO;
-    /// Boolean to flag if blueprint should be written out
-    bool blueprintDot = false;
+    /// Chose to emulate the building and plot a graph instead with the given
+    /// name
+    std::string emulateToGraph = "";
```

Comment:
```suggestion
    /// Choose to emulate the building and plot a graph instead with the given
    /// name
    std::string emulateToGraph = "";
```

Can you elaborate what "emulate" means here? Do you mean that it will **only** build the graph and not run the construction itself?

The graph writing works in the same blueprint tree that is used for construction, right?

Reply:
Exactly, it will not do the building, because,  e.g. if the building throws and exception, this will show you how/where.

Reply:
I will clarify in the comments!

Reply:
I clarified with:

```
    /// If this string is not empty, the detector is not built, but only
    /// a building graph is generated as `.dot` file with the given name
    std::string emulateToGraph = "";
```